### ISSUE TITLE
Comments replaced by block comments inside Markdown code block

### ISSUE THREAD

**Issue by kuukunen** [doxygen/doxygen#7570]
(Windows 10, Doxygen version 1.8.17 (b5fa3cd1c6e6240e20d3b80a70e3f04040b32021*))

I had a weird issue where Doxygen seems to ignore almost everything inside a &grave;&grave;&grave; markdown code block in a .md documentation file... except it somehow replaces multiline /// comments with /** block comments */

Basically in my CodeConvention.md file this:

&grave;&grave;&grave;
/// Brief explanation of a function.
/// More explanation.
void Function1();

/// FF.
/// FF.
/&ast;&ast;\verbatim
Verbatim block
\endverbatim&ast;/
void Function2();

/// Brief explanation of a function.
void Function3();

/// Brief explanation of a function.
/// More explanation.
void Function4();
&grave;&grave;&grave;

becomes:

```
/// Brief explanation of a function.
/// More explanation.
void Function1();
 
/// FF.
/// FF.
/**\verbatim
Verbatim block
\endverbatim*/
void Function2();
 
/// Brief explanation of a function.
void Function3();
 
/** Brief explanation of a function.
 *  More explanation. */
void Function4();
```

Note the comments before Function4.

This seems to happen to multiline comments that appear after a /** block comment */ (The part with \verbatim, in this case, but doesn't need any commands inside)

**Comment by kuukunen** [comment]
I don't know if this should be a completely new issue, but the block comment also somehow breaks aliases.

With:  `ALIASES                += ns{1}="\ref NS::\1 \"\1\""`

This:

---

\ref NS::Function
\ns{Function}

&grave;&grave;&grave;
/// something
&grave;&grave;&grave;

\ref NS::Function
\ns{Function}

&grave;&grave;&grave;
/** something */
&grave;&grave;&grave;

\ref NS::Function
\ns{Function}

---

Becomes:

[NS::Function](dummy_link_to_Function) [Function](dummy_link_to_Function)

```
/// something
```

[NS::Function](dummy_link_to_Function) [Function](dummy_link_to_Function)

```
/** something */
```

[NS::Function](dummy_link_to_Function) \ns{Function}

**Comment by kuukunen** [comment]
On a bit further investigation, it seems this is a duplicate of #7117

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #7572, for the 2 mentioned problems.

These 2 problems have a common ground, but are not a duplicate of #7117, #7117 does not have fenced code blocks but has direct `*/` etc.

**Comment by kuukunen** [comment]
I checked and seems like those commits fix both issues for me. Awesome!
Thank you very much for quick response!

**Comment by albert-github** [comment]
(Updated) Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.18.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
